src/applications/lgy/coe/form/containers/introduction-content/COEIntroPageBox.jsx

```diff
@@ -11,7 +11,7 @@ const COEIntroPageBox = props => {
   if (props.coe.status) {
     switch (props.coe.status) {
       case COE_ELIGIBILITY_STATUS.available:
-        return <COEAvailable />;
+        return <COEAvailable downloadURL={props.downloadURL} />;
```

Comment:
This is a really small thing, and not worth holding the PR up for, but we could destructure these props to make this a little cleaner, thoughts?

Reply:
Yeah I honestly thing that this is valid in a few other components as well. I'd like to refactor those bits when I come back to do error handling

Reply:
@jerekshoe For sure, lets make a ticket for refactoring this and we can come back to it later